modules/network-observability-configuring-custom-metrics.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-configuring-custom-metrics_{context}"]
+= Configuring custom metrics using FlowMetrics API
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *FlowMetrics*.
+. Use the *Project:*  dropdown list to select the project of the Network Observability Operator instance. 
+. Click *Create FlowMetric*.
+. Here are a couple of examples of `FlowMetrics` configurations you can add:
```

Comment:
This doesn't seem like a step of procedure. You may want to move it to a different section.

Reply:
Ok I can agree with this. I adjusted the step language to instead say: "Configure the `FlowMetrics` resource, such as the following examples" then there are a couple of examples you can use to add configurations.